.classpath

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We don't want to check in Eclipse specific stuff in the fixtures. See comment to `.gitignore`